chainer/functions/activation/leaky_relu.py

```diff
@@ -43,15 +43,31 @@ def backward_gpu(self, x, gy):
 def leaky_relu(x, slope=0.2):
     """Leaky Rectified Linear Unit function.
 
-    This function is expressed as :math:`f(x) = \\max(x, ax)`, where :math:`a`
-    is a configurable slope value.
+    .. math:: f(x)=\\max(x, ax),
+
+    where :math:`a` is a configurable slope value.
 
     Args:
-        x (~chainer.Variable): Input variable.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable. A :math:`(s_1, s_2, ..., s_n)`-shaped float array.
         slope (float): Slope value :math:`a`.
 
     Returns:
-        ~chainer.Variable: Output variable.
+        ~chainer.Variable: Output variable. A
+        :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
+    .. admonition:: Example
+
+        >>> x = np.array([[-1, 0], [2, -3], [-2, 1]], 'f')
+        >>> x
+        array([[-1.,  0.],
+               [ 2., -3.],
+               [-2.,  1.]], dtype=float32)
+        >>> F.leaky_relu(x).data
```

Comment:
How about specifying `slope=0.2` explicitly here to show the results surely use `a = 0.2` in the above equation?